core/src/test/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitorTest.java

```diff
@@ -329,7 +330,121 @@ public void should_refresh_node_list_from_local_and_peers() {
   }
 
   @Test
-  public void should_stop_executing_queries_once_closed() throws Exception {
+  public void should_refresh_node_list_from_local_and_peers_v2() {
```

Comment:
This test is not directly related to peer checks, but It was missing so I added it.